loadFolders.xml

```diff
@@ -0,0 +1,6 @@
+<loadFolders>

```

Comment:
```
    <v1.0>
        <li>/</li>
    </v1.0>
```
Should be included, and we can remove the 1.1 load folder entirely, as the same Defs will be loaded in both versions.